src/schemas/recording/index.ts

```diff
@@ -1,2 +1,47 @@
-export * from './v1/browser'
-export { Recording } from './v1/recording'
+import type { Page } from 'har-format'
+import { z } from 'zod'
+
+import { EntryWithOptionalResponse } from '@/types/har'
+
+import * as v1 from './browser/v1'
+import * as v2 from './browser/v2'
```

Comment:
@allansson I think you suggestion makes a lot of sense, because we're much less likely to change the schema of the HAR log. I added the changes and went a few steps further: created the `browser` sub-folder with folders for each version and move the browser event schemas there - wdyt?